node/cmd/guardiand/adminserver.go

```diff
@@ -210,6 +210,18 @@ func wormchainInstantiateContract(req *nodev1.WormchainInstantiateContract, time
 	return v, nil
 }
 
+// wormchainMigrateContract converts a nodev1.WormchainMigrateContract to its canonical VAA representation
+func wormchainMigrateContract(req *nodev1.WormchainMigrateContract, timestamp time.Time, guardianSetIndex uint32, nonce uint32, sequence uint64) (*vaa.VAA, error) {
+	instantiationParams_hash := vaa.CreateMigrateCosmwasmContractHash(req.CodeId, req.Contract, []byte(req.InstantiationMsg))
+
+	v := vaa.CreateGovernanceVAA(timestamp, nonce, sequence, guardianSetIndex,
+		vaa.BodyWormchainInstantiateContract{
```

Comment:
Why is this using BodyWormchainInstantiateContract

Reply:
good catch!